config/config/preferences.yml

```diff
@@ -67,3 +67,9 @@ auto_follow_comment_threads:
   description: >
     Automatically follow any comment thread you participate in.
   default: 'true'
+
+sticky_header:
+  type: boolean
+  description: >
+    Make the Codidact navigation bar sticky
```

Comment:
Can we change this description to "Make the top navigation bar sticky."? With all the work that's being done to remove Codidact-specific things, I want to make sure we're not adding more now.

Reply:
Done 👍